amy/templates/forms/trainingrequest.html

```diff
@@ -8,52 +8,19 @@
 
 {% block content %}
 
-<p class="lead">
-  Our instructor training classes are for people who wish to learn how to
-  teach Carpentry workshops. We teach impactful pedagogical techniques and
-  prepare you to become an Instructor in our community. We use this form to
-  register interest and build new communities of Instructors around the world.
+<p>
+  This form serves as the entry point for Instructor Training for two groups:
 </p>
 
-<p>If you wish to apply to our open instructor training program, please
-fill in the form below. Tell us about yourself, what excites you about
-teaching, and how The Carpentries can help in your community.
-Spaces are limited, so use this form to share with us what excites you
-about spreading our teaching to your organization. We will notify
-applicants as limited spaces become available. If you have any
-questions, please mail <a href="mailto:team@carpentries.org">
-team@carpentries.org</a>.</p>
+<b>For trainees selected by Carpentries member organizations or groups that have arranged for training</b>, it serves as a trainee <b>Profile Creation form</b>. This creates an entry in our database and allows us to record attendance and track your progress through the Instructor certification process. Form responses are not reviewed in detail but may be used in programmatic assessment. Please note that this form is separate from event registration. Your site contact will provide you with instructions to register for an Instructor Training event. In response to the first question, please select <i>Profile Creation for Pre-approved Trainees</i> and enter in your provided registration code.  If you do not have a registration code, please enter the name of the institution you are joining through.</p>
```

Comment:
`<p>` missing at the beginning of this paragraph.

Reply:
Fixed